lib/logstash/outputs/newrelic.rb

```diff
@@ -64,34 +65,44 @@ def time_to_logstash_timestamp(time)
     end
   end
 
-  def encode(event_hash)
-    log_message_hash = {
-      # non-intrinsic attributes get put into 'attributes'
-      :attributes => event_hash
-    }
+  def to_NR_logs(logstash_events)
```

Comment:
Minor but very conventional in Ruby to use all lower-case in method names so the `NR` here looks weird.

Reply:
Fixed!